### ISSUE TITLE
2 random results for same package

### ISSUE THREAD

**Issue by sda399** [anchore/syft#1464]
Hi,
awsome software thx for sharing.

I have random results with this package, sometimes reported with 2 deps, is there a option to fix this behavior?

```
> syft --version
syft 0.66.1
> curl -O 'https://repo1.maven.org/maven2/net/bytebuddy/byte-buddy/1.11.22/byte-buddy-1.11.22.jar'
> for 1 in $(seq 10); do syft byte-buddy-1.11.22.jar; done;
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [1 packages]

NAME            VERSION  TYPE         
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [1 packages]
NAME            VERSION  TYPE         
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [1 packages]
NAME            VERSION  TYPE         
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [2 packages]
NAME            VERSION  TYPE         
byte-buddy      1.11.22  java-archive  
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [2 packages]
NAME            VERSION  TYPE         
byte-buddy      1.11.22  java-archive  
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [2 packages]
NAME            VERSION  TYPE         
byte-buddy      1.11.22  java-archive  
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [1 packages]

NAME            VERSION  TYPE         
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [1 packages]

NAME            VERSION  TYPE         
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [2 packages]

NAME            VERSION  TYPE         
byte-buddy      1.11.22  java-archive  
byte-buddy-dep  1.11.22  java-archive  
 ✔ Indexed byte-buddy-1.11.22.jar 
 ✔ Cataloged packages      [1 packages]
NAME            VERSION  TYPE         
byte-buddy-dep  1.11.22  java-archive
```

**Comment by sda399** [comment]
Hi thx for early reply, I noticed it on Ubuntu 22 x86_64 and Debian 11 (bullseye) docker images

**Comment by kzantow** [comment]
Hi, and apologies for the delay here. There was an issue identified causing nondeterministic behavior in Java archive identification, which was fixed in Syft 1.10.0. I'm going to close this issue as I am no longer able to reproduce the nondeterministic behavior, but please do let us know if you still have the issue!
